transports/upgrades/sqlserver-unicode-headers.md

```diff
@@ -0,0 +1,64 @@
+---
+title: SQL Server Transport Upgrade - Supporting Unicode in headers
+summary: Instructions on how to add support for Unicode characters in message headers
+component: SqlTransport
+reviewed: 2017-07-17
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 5
+ - 6
+---
+
+
+## Summary
+
+This document explains how to patch a system to [support Unicode characters in message headers using the SQL Server transport](https://github.com/Particular/NServiceBus.SqlServer/issues/340). The issue may cause data loss when using header names or values that contain Unicode characters.
```

Comment:
not sure if `Unicode characters` is precise enough term for the doco. I think the problem is usage of characters outside of the set supported by the selected DB's collation.